src/librustc/diagnostics.rs

```diff
@@ -1886,7 +1886,51 @@ This explicitly states that you expect the trait object `SomeTrait` to
 contain references (with a maximum lifetime of `'a`).
 
 [1]: https://github.com/rust-lang/rfcs/pull/1156
-"##
+"##,
+
+E0454: r##"
```

Comment:
Can’t this be generalised to the same E0452? I mean there seems really little use to differentiate between an empty name and a malformed name.